_data/compatibility/bitrefill.yaml

```diff
@@ -0,0 +1,65 @@
+---
+name: Bitrefill
+internal_url: /en/compatibility/bitrefill
+logo: /img/compatibility/bitrefill/bitrefill.png
+rbf:
+  tested:
+    date: "2018-11-06"
+    platforms:
+      - web
+    version: "n/a"
+  features:
+    receive:
+      notification: "na"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "true"
+    send:
+      signals_bip125: "true"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "untested"
```

Comment:
@harding the interesting case here is that bitrefill send RBF, but as a user, I cannot create a replacement transaction. I think the text for "na" makes more sense here, so I am switching these to "na":

> Because no transaction replacement is possible, we could not test whether original or replacement sent transactions are shown after replacement.

Reply:
Maybe "No transaction replacement is possible **in the user interface, so** we could not test whether original or replacement sent transactions are shown after replacement."